conu/backend/docker/image.py

```diff
@@ -220,6 +220,19 @@ def push(self, repository=None, tag=None):
                     raise ConuException("There was an error while pushing the image %s: %s",
                                         self.name, error)
         return image
+    
+    def copy(self, repository, tag="latest"):
+        """
+        WIP
+        Copy image to repository:tag
+        :param repository to be copied to
+        :param tag
+        :return: the new DockerImage later
+        """
+        subprocess.call(["skopeo", "copy",
+                         "docker://" + self.name + ":" + self.tag,
+                         "docker://" + repository + ":" + tag])
```

Comment:
the transport here should definitely be configurable (I know it's WIP, just sayin' up front)

Reply:
ok, working on it